types/src/keyless/proof_simulation.rs

```diff
@@ -158,21 +207,16 @@ impl<E: Pairing, QAP: R1CSToQAP> Groth16Simulator<E, QAP> {
     }
 }
 
-
-
 /// Generates a trapdoor proving and verifiying key pair intended for proof simulation, in addition to a vector of public inputs, from
 /// circom-generated .r1cs and .wasm files, and a .json file containing the public inputs. To be
 /// used to update `test_prove_and_verify` after circuit changes occur
 /// WARNING: The files referenced in this function are not present in this repo and msut be added
 /// manually
-// TODO: There is a version conflict between this piece of code and `aptos-dkg` with the
-// `num-bigint` dependency. Ideally we resolve this by updating `aptos-dkg` to `num-bigint` version
-// `0.4.5`
-/*#[allow(dead_code)]
-fn generate_keys_and_inputs<E: Pairing>() {
+#[allow(dead_code)]
+fn generate_keys_and_inputs<E: Pairing>(wasm_file_path: String, r1cs_file_path: String) {
     let cfg = CircomConfig::<E>::new(
-        "./circuit-files/keyless_main.wasm",
-        "./circuit-files/keyless_main.r1cs",
+        wasm_file_path,
+        r1cs_file_path,
     ).unwrap();
     let mut builder = CircomBuilder::new(cfg);
     let mut input_file = File::open("./circuit-files/keyless_input.json").unwrap();
```

Comment:
We can't have hardcoded paths like this in the code...

Reply:
Oops missed that one, will fix

Reply:
Done